src/lib/homestore_backend/hs_blob_manager.cpp

```diff
@@ -362,15 +362,18 @@ void HSHomeObject::on_blob_del_commit(int64_t lsn, sisl::blob const& header, sis
     blob_info.blob_id = *r_cast< blob_id_t* >(key.bytes);
 
     auto r = move_to_tombstone(index_table, blob_info);
-    if (!r) {
+    if (!r && recovery_done_) {
```

Comment:
Is it possible to verify it with solorepldev?

Reply:
Currently we dont have a way to inject error.